core-bundle/src/Resources/contao/drivers/DC_Folder.php

```diff
@@ -2597,6 +2597,7 @@ protected function generateTree($path, $intMargin, $mount=false, $blnProtected=t
 			$md5 = substr(md5($folders[$f]), 0, 8);
 			$content = scan($folders[$f]);
 			$currentFolder = StringUtil::stripRootDir($folders[$f]);
+            $objFolder = FilesModel::findByPath($currentFolder);
```

Comment:
We should probably check for `$this->blnIsDbAssisted` here.

```suggestion
            $objFolder = $this->blnIsDbAssisted ? FilesModel::findByPath($currentFolder) : null;
```